web/src/features/moreCast2/components/GridComponentRenderer.tsx

```diff
@@ -36,19 +39,52 @@ export class GridComponentRenderer {
   ) => {
     return <ForecastHeader colDef={params.colDef} columnClickHandlerProps={columnClickHandlerProps} />
   }
-  public renderHeaderWith = (params: GridColumnHeaderParams) => {
+  public renderHeaderWith = (
+    params: Pick<GridColumnHeaderParams, 'field'> & {
+      colDef: Pick<GridColumnHeaderParams['colDef'], 'field' | 'headerName'>
+    },
+    allRows?: MoreCast2Row[]
+  ) => {
+    const headerName = params.colDef.headerName ?? ''
+
     if (params.field.endsWith('_BIAS')) {
-      const headerName = params.colDef.headerName ?? ''
       const index = headerName.indexOf('_BIAS')
       const prefix = headerName.slice(0, index)
       return (
         <div data-testid={`${params.colDef.field}-column-header`}>
-          <Box sx={{ height: '1rem' }}>{prefix}</Box>
-          <Box>bias</Box>
+          <Typography data-testid={`${params.colDef.field}-column-header`} style={{ fontSize: '14px' }}>
+            {prefix}
+          </Typography>
+          <Typography style={{ fontSize: '14px' }}>bias</Typography>
+        </div>
+      )
+    }
+    const modelType = weatherModelsWithTooltips.find(model => headerName === model)
+    if (modelType && allRows && allRows.length > 0) {
+      const timestampField = `predictionRunTimestamp${modelType}` as keyof MoreCast2Row
+      const timestamp = allRows[0][timestampField] as string | null | undefined
```

Comment:
It looks like `allRows` are ordered by ascending date, so `allRows[0][timestampField]` gives us an old model run date.  I think you'll need to search for the most recent date for the specific model in all the rows.